libraries/app/src/index.js

```diff
@@ -124,6 +127,29 @@ const app = async (options) => {
     });
   }
 
+  app.use('/__version__', (req, res) => {
+    const taskclusterVersionFile = path.resolve(rootdir.get(), 'version.json');
+
+    try {
+      const taskclusterVersion = fs.readFileSync(taskclusterVersionFile).toString().trim();
+      res.header('Content-Type', 'application/json');
+      res.send(taskclusterVersion);
+    } catch (err) {
+      res.header('Content-Type', 'application/json');
+      res.status(500).send({ error: 'Not found' });
+    }
+  });
+
+  app.use('/__heartbeat__', (req, res) => {
+    res.header('Content-Type', 'application/json');
+    res.status(200).send({});
+  });
```

Comment:
Should the heartbeat endpoint replicate the logic in ping.js? https://github.com/taskcluster/taskcluster/blob/4fadfb8527ec4fecb50145f679f9641ed23eca1c/infrastructure/tooling/src/smoketest/checks/ping.js#L40-L63

Reply:
It kind of serves the same purpose, but I don't think that uptime information is especially useful.  Just returning an empty body is probably enough.